llm/llama.h

```diff
@@ -0,0 +1,1227 @@
+/**
```

Comment:
possible to update the generate scripts to copy this file in after the patches have been applied?